lib/web/apiserver.go

```diff
@@ -350,6 +350,8 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 	h.POST("/webapi/mfa/login/finish", httplib.MakeHandler(h.mfaLoginFinish))
 	h.POST("/webapi/mfa/login/finishsession", httplib.MakeHandler(h.mfaLoginFinishSession))
 	h.GET("/webapi/mfa", h.WithAuth(h.getMFADevicesHandle))
+	h.POST("/webapi/mfa/token/:token/authnchallenge", httplib.MakeHandler(h.createAuthenticateChallengeWithTokenHandle))
```

Comment:
Spell it out? Same below.

```suggestion
	h.POST("/webapi/mfa/token/:token/authenticatechallenge", httplib.MakeHandler(h.createAuthenticateChallengeWithTokenHandle))
```